src/main/java/org/folio/event/handler/HoldingUpdateAsyncRecordHandler.java

```diff
@@ -57,7 +58,18 @@ protected Future<Void> processInventoryUpdateEvent(ResourceEvent resourceEvent,
       log.info("processInventoryUpdateEvent:: No instance id or search location ids to update in holding '{}', ignoring update", holder.getHoldingId());
       return Future.succeededFuture();
     }
-    return processHoldingUpdateEvent(holder);
+    return consortiumConfigurationService.getCentralTenantId(getContext(), headers)
+      .compose(centralTenantId -> {
+        holder.setCentralTenantId(centralTenantId);
+        return processHoldingUpdateEvent(holder)
+          .compose(poLines -> {
+            if (Boolean.FALSE.equals(holder.isPoLinesUpdated()) && Objects.nonNull(centralTenantId)) {
```

Comment:
if possible to return future with booelan value for processHoldingUpdateEvent() it would be easier and eliminate poLinesUpdated property in holder

Reply:
okay, if we could not rely on return value, lets remain as it

Reply:
> okay, if we could not rely on return value, lets remain as it (its like contracts for updating rows in DB, if some rows updated Spring Data returns true or false otherwsie)

I will check the affected rows after POL update tomorrow